src/gui/qgsadvanceddigitizingdockwidget.h

```diff
@@ -67,9 +69,9 @@ class GUI_EXPORT QgsAdvancedDigitizingDockWidget : public QgsDockWidget, private
     Q_FLAG( CadCapacities )
 
     /**
-     * Additional constraints which can be enabled
+     * Between line constraints which can be enabled
      */
-    enum class AdditionalConstraint SIP_MONKEYPATCH_SCOPEENUM : int
+    enum class BetweenLineConstraint SIP_MONKEYPATCH_SCOPEENUM : int
```

Comment:
that's an API break.
You might consider moving the enum to qgis.h (see for instance https://github.com/qgis/QGIS/pull/43314 and using SIP_MONKEYPATCH_SCOPEENUM_UNNEST would solve the API break issue)

Reply:
also the name doesn't sound really clear to me. Why between?

Reply:
Yes, this modification changes the API (in addition to the enum, there is the `additionalConstraint` method). I did it anyway because besides thinking that it was important to reduce confusion related to my proposal and future potential ones, I think (maybe wrongly) that it's very little used in plugins.

About the name, I admit that I put the first thing that came to mind and thinking of changing it later. If you have an idea, I'm interested!

Reply:
By correctly using SIP_MONKEYPATCH_SCOPEENUM_UNNEST, there will be no API break.

`CadConstraint` is bit more explicit to me than `BetweenLineConstraint`. I don't have a better proposition.